mlflow/store/artifact/azure_blob_artifact_repo.py

```diff
@@ -60,20 +60,21 @@ def parse_wasbs_uri(uri):
         parsed = urllib.parse.urlparse(uri)
         if parsed.scheme != "wasbs":
             raise Exception("Not a WASBS URI: %s" % uri)
-        match = re.match("([^@]+)@([^.]+)\\.blob\\.core\\.windows\\.net", parsed.netloc)
+        match = re.match("([^@]+)@([^.]+)\\.(blob\\.core\\.(windows\\.net|chinacloudapi\\.cn))", parsed.netloc)
```

Comment:
Can we refactor this line to use a raw string?

```suggestion
        match = re.match(r"([^@]+)@([^.]+)\.(blob\.core\.(windows\.net|chinacloudapi\.cn))", parsed.netloc)
```

Reply:
Can we also fix the docstring of this function?

Reply:
fixed.